views/about.erb

```diff
@@ -0,0 +1,70 @@
+<html>
+<header>
```

Comment:
Whenever you start nesting tags you want to use indentation to denote that the tags are nested within one another